python/lsst/meas/astrom/matcher_probabilistic.py

```diff
@@ -99,12 +143,136 @@ class MatchProbabilisticConfig(pexConfig.Config):
         doc='The target table column for the second spatial coordinate (usually y or dec).'
             'Units must match column_ref_coord2.',
     )
+    coords_spherical = pexConfig.Field(
+        dtype=bool,
+        default=True,
+        doc='Whether column_*_coord[12] are spherical coordinates (ra/dec) or not (pixel x/y)',
+    )
+    coords_ref_factor = pexConfig.Field(
+        dtype=float,
+        default=1.0,
+        doc='Multiplicative factor for reference catalog coordinates.'
+            'If coords_spherical is true, this must be the number of degrees per unit increment of '
+            'column_ref_coord[12]. Otherwise, it must convert the coordinate to the same units'
+            ' as the target coordinates.',
+    )
+    coords_target_factor = pexConfig.Field(
+        dtype=float,
+        default=1.0,
+        doc='Multiplicative factor for target catalog coordinates.'
+            'If coords_spherical is true, this must be the number of degrees per unit increment of '
+            'column_target_coord[12]. Otherwise, it must convert the coordinate to the same units'
+            ' as the reference coordinates.',
+    )
+    coords_ref_to_convert = pexConfig.DictField(
+        default=None,
+        keytype=str,
+        itemtype=str,
+        dictCheck=lambda x: len(x) == 2,
+        doc='Dict mapping sky coordinate columns to be converted to pixel columns',
+    )
+    mag_zeropoint_ref = pexConfig.Field(
+        dtype=float,
+        default=31.4,
+        doc='Magnitude zeropoint for reference catalog.',
+    )
+
+    def format_catalogs(
+        self,
+        catalog_ref: pd.DataFrame,
+        catalog_target: pd.DataFrame,
+        select_ref: np.array = None,
+        select_target: np.array = None,
+        radec_to_xy_func: Callable = None,
+        return_converted_columns: bool = False,
+        **kwargs,
+    ):
+        """Format matched catalogs that may require coordinate conversions.
+
+        Parameters
+        ----------
+        catalog_ref : `pandas.DataFrame`
+            A reference catalog for comparison to `catalog_target`.
+        catalog_target : `pandas.DataFrame`
+            A target catalog with measurements for comparison to `catalog_ref`.
+        select_ref : `numpy.ndarray`, (Nref,)
+            A boolean array of len `catalog_ref`, True for valid match candidates.
+        select_target : `numpy.ndarray`, (Ntarget,)
+            A boolean array of len `catalog_target`, True for valid match candidates.
+        radec_to_xy_func : `typing.Callable`
+            Function taking equal-length ra, dec arrays and returning an ndarray of
+            - ``x``: current parameter (`float`).
+            - ``extra_args``: additional arguments (`dict`).
+        return_converted_columns : `bool`
+            Whether to return converted columns in the `coord1` and `coord2`
+            attributes, rather than keep the original values.
+        kwargs
+
+        Returns
+        -------
```

Comment:
Missing the things

Reply:
This was fixed in DM-33949 (https://github.com/lsst/meas_astrom/commit/b9f4f1e462b4e517a11dd037a1637d1ea19faa21), which superseded this entire PR.